src/obu.rs

```diff
@@ -2442,14 +2439,6 @@ unsafe fn parse_obus(
                     if payload_size <= 0 {
                         writeln!(c.logger, "Malformed ITU-T T.35 metadata message format");
                     } else {
-                        let r#ref = rav1d_ref_create(
-                            ::core::mem::size_of::<DRav1d<Rav1dITUTT35, Dav1dITUTT35>>()
-                                + payload_size as usize * ::core::mem::size_of::<u8>(),
-                        );
-                        if r#ref.is_null() {
-                            return Err(ENOMEM);
-                        }
```

Comment:
Was this code just creating a ref and then doing nothing with it? Do you know if there was some case where that `ref` var was being used originally?

Reply:
Yup, doing nothing.  It didn't get flagged as dead code due to the error checking.  It was originally being used but I replaced it with an `Arc`.  So actually this is a memory leak I missed.